src/routes/demos/.form/.Footer.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+  import { formStore } from '$lib/store';
+  export let displayFootnote: boolean;
```

Comment:
Please see previous comment on naming boolean variables.

Reply:
This is a property registered inside Demo entity. Display/show/has they all refer to the same thing. I did change the other ones though.